Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.h

```diff
@@ -29,9 +29,7 @@ typedef struct _MSFT_nxOMSAutomationWorkerResource /* extends OMI_BaseResource *
     MI_Instance __instance;
     /* OMI_BaseResource properties */
     /* MSFT_nxOMSAutomationWorkerResource properties */
-    /*KEY*/ MI_ConstStringField WorkspaceId;
```

Comment:
If the JSON string contains the workspace ID, then it will be unique on a machine as a key with multiple workspaces. I believe that when DSC resources support multi-homing, they will need to utilize the workspace ID or something including it as a key.

So no, this is not a concern from the current design perspective.